tayoto_mikaellaantonette/index.html

```diff
@@ -86,28 +86,28 @@ <h2>My Goals</h2>
           <form class="comment">
             <div>
               <h2>Comments</h2>
-              <label for="name">Name: </label>
-              <input type="text" id="name" oninput="name()" />
-            </div>
-            <div>
-              <p>Comment:</p>
-              <textarea
-                name="gcomment"
-                id="gcomment"
-                oninput="comment()"
-                cols="30"
-                rows="10"
-              ></textarea>
-            </div>
-            <div>
-              <button
-                class="submit-goal"
-                type="button"
-                id="comment_button"
-                disabled
-              >
-                Submit
-              </button>
+              <section class="comment">
+                <div>
+                  <label for="name">Name:</label>
+                  <input type="text" id="name" required />
+                </div>
+                <div>
+                  <label for="gcomment">Comment:</label>
+                  <textarea id="gcomment"></textarea>
+                </div>
+                <div>
+                  <button type="submit" 
+                  class="submit-goal" 
+                  id="comment_button">
+                    Submit
+                  </button>
+                </div>
+                <div>
+                  <button id="sort-asc">Sort by Date (ASC)</button>
+                  <button id="sort-desc">Sort by Date (DESC)</button>
```

Comment:
ID should be written in snake case.

Reference:
https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit#slide=id.g24da8de92d7_0_5